spago.lock

```diff
@@ -0,0 +1,1170 @@
+{
```

Comment:
I cloned this PR branch and ran `spago build` and it created a completely different `spago.lock` file. I wonder why? I'm on spago v0.93.37.

Reply:
The `spago.lock` file is only for development of this package anyway, is my understanding...

Reply:
```
$ rm -fdr output && spago build --censor-stats --strict --ensure-ranges --pedantic-packages
no changes

 ~/projects/contrib/purescript-uint   update-spago-next  spago --version
0.93.40

```

Reply:
The CI passed with spago v0.93.40 https://github.com/purescript-contrib/purescript-uint/actions/runs/11305240509/job/31444528947

Reply:
Ok I see now that the `spago.lock` file format has changed to JSON for spago v0.93.40...